.env

```diff
@@ -2,6 +2,22 @@ CLEARING_HOUSE=http://109.232.32.193:8280/data
 BROKER_URL=https://broker.ids.isst.fraunhofer.de/infrastructure
 
 DISABLE_SSL_VALIDATION=true
+#Enable HTTPS
+SERVER_SSL_ENABLED=true
+#SSL settings
+KEYSTORE_NAME=changeit(JKS format)
+KEY_PASSWORD=changeit
+KEYSTORE_PASSWORD=changeit
+ALIAS=changeit
+#AISEC DAPS settings (used by IDSCP2)
+DAPS_KEYSTORE_NAME=eng-keystore.jks
+DAPS_KEYSTORE_PASSWORD=password
+DAPS_KEYSTORE_ALIAS=1
+DAPS_TRUSTORE_NAME=server.truststore.p12
```

Comment:
This one is more general truststore, so maybe to move it next to "general" jks property